configs/components/pxp-agent.rb

```diff
@@ -103,7 +103,9 @@
   when "aix"
     pkg.install_service "resources/aix/pxp-agent.service", nil, "pxp-agent"
   when "windows"
-    puts "Service files not enabled on windows"
+    # Note - this definition indicates that the file should be filtered out from the Wix
+    # harvest. A corresponding service definition file is also required in resources/windows/wix
+    pkg.install_service "SourceDir\\ProgramFiles64Folder\\PuppetLabs\\Puppet\\bin\\nssm.exe"
```

Comment:
We don't have any variables that help with the other paths, since they're getting pulled from the local repos. Since we're installing `nssm.exe`, we know where it's going to go already (https://github.com/puppetlabs/puppet-agent/blob/master/configs%2Fcomponents%2Fnssm.rb#L11). In this case, we're installing nssm to bindir. Bindir currently evaluates to `C:/ProgramFiles64Folder/PuppetLabs/Puppet/puppet/bin`, so with a little path munging, if we ever change the location of nssm.exe, then we'd only have to change it in one place rather than three.